src/app/components/team/RoleSelect.js

```diff
@@ -41,40 +37,35 @@ const RoleSelect = ({
     : roles;
 
   return (
-    <FormControl
-      variant="outlined"
-      style={{ minWidth: units(20) }}
-      fullWidth={fullWidth}
+    <Select
+      className="role-select"
+      label={showLabel ? <FormattedMessage id="roleSelect.selectLabel" defaultMessage="Workspace permission" description="Label for select input for user to choose the permissions role for this user" /> : null}
+      disabled={disabled}
+      required={showLabel}
+      onChange={onChange}
+      value={value}
     >
-      <Select
-        className="role-select"
-        disabled={disabled}
-        input={<OutlinedInput name="role-select" labelWidth={0} />}
-        onChange={onChange}
-        value={value}
-      >
-        {
-          filteredRoles.map(r => (
-            <MenuItem className={`role-${r.value}`} key={r.value} value={r.value}>
-              {r.label}
-            </MenuItem>
-          ))
-        }
-      </Select>
-    </FormControl>
+      {
+        filteredRoles.map(r => (
+          <option className={`role-${r.value}`} key={r.value} value={r.value}>
+            {r.label}
+          </option>
+        ))
+      }
+    </Select>
   );
 };
 
 RoleSelect.defaultProps = {
   disabled: false,
   excludeRoles: [],
-  fullWidth: false,
+  showLabel: false,
```

Comment:
No need for the full width anymore, but this show label is to hide the label when in the table view